client/src/components/TrackGroup/Wishlist.tsx

```diff
@@ -28,10 +29,20 @@ const Wishlist: React.FC<{ trackGroup: TrackGroup }> = ({ trackGroup }) => {
   return (
     <Button
       onClick={onClick}
-      compact
-      variant="link"
       className={css`
         color: inherit !important;
+        :hover {
```

Comment:
Does this work? This doesn't have to be &:hover?

```suggestion
        &:hover {
```

Reply:
Yeah it worked without the &

Reply:
I guess I don't understand that 😬